CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog for dx-toolkit releases
+
+This change log spiritually adheres to [these guidelines](http://keepachangelog.com/en/0.3.0/).
+
+Categories for each release: Added, Changed, Deprecated, Removed, Fixed, Security
+
+
+## [Unreleased]
```

Comment:
Yes.  This is one of the suggestions to "minimize the effort required" to maintain the Change log buried somewhere in the user guide above.  The idea is when the release is cut, the "Unreleased" is given a version number.